### DISCUSSION TITLE
Spotlight for basic search input

### DISCUSSION THREAD

**Discussion by erikdemarco** [mantinedev/mantine#6866] [Q&A]
because its not mentioned in the docs, i'm curious Is it possible to use spotlight component that act like  basic search input (see google.com). so when user typed it will show keyword list (for example example called from http://suggestqueries.google.com/complete/search?client=firefox&q=YOURQUERY). and when enter is pressed, it will navigate to /search?q=YOURQUERY

**Comment by rtivital** [comment]
You can do that with Autocomplete component – https://mantine.dev/core/autocomplete/
